pombola/south_africa/views/download.py

```diff
@@ -0,0 +1,86 @@
+import os
+
+from django.db.models import Prefetch, Q
+from django.http import Http404, StreamingHttpResponse
+from django.shortcuts import get_object_or_404
+from django.views.generic import TemplateView
+
+import xlsx_streaming
+from pombola.core.models import Contact, Organisation, Person, Position
+
+MP_DOWNLOAD_TEMPLATE_SHEET = os.path.join(
+    os.path.dirname(os.path.realpath(__file__)), "mp-download-template.xlsx"
+)
+
+
+def get_email_address_for_person(person):
+    if person.email:
+        return person.email
+    if len(person.email_addresses) > 0:
+        return person.email_addresses[0].value
```

Comment:
let's return all email addresses separated by a space, or has someone asked that we only return the first?

Reply:
Nope, they didn't specify. Will do :+1: